neo/Prompt/Commands/Invoke.py

```diff
@@ -35,7 +35,7 @@
 from neo.VM.OpCode import *
 import json
 
-DEFAULT_MIN_FEE = Fixed8.FromDecimal(.001)
+DEFAULT_MIN_FEE = Fixed8.FromDecimal(.0001)
```

Comment:
@localhuman, what's your level of confidence that this is a sufficient fee? In my experience, I had a hard time getting every relayed transaction accepted when I used `.0001` or even `.0002`. When running a large number of automated transactions, I'd see some transactions not go through successfully. Only once I bumped it up to `.0004` was I able to consistently get all relayed transactions to get processed successfully.